richtext-ui/src/jvmMain/kotlin/com/halilibo/richtext/ui/previews/BlockQuotePreview.kt

```diff
@@ -0,0 +1,41 @@
+package com.halilibo.richtext.ui.previews
+
+import androidx.compose.desktop.ui.tooling.preview.Preview
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Box
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.CompositionLocalProvider
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import com.halilibo.richtext.ui.BlockQuote
+import com.halilibo.richtext.ui.LocalInternalContentColor
+import com.halilibo.richtext.ui.RichTextScope
+import com.halilibo.richtext.ui.Text
+
+@Preview
+@Composable
+private fun BlockQuotePreviewOnWhite() {
```

Comment:
Why move these to a separate file? Doesn't that defeat the purpose a bit? (That is, being the ability to see the component rendered on the same screen as you work on it.)

Reply:
That definitely does defeat the purpose. However, previews from common module do not work in neither Android Studio nor IDEA. The closest we can get is placing previews under jvm source set.

Reply:
Oh, damn.